tests/opf/test_io.py

```diff
@@ -126,12 +132,13 @@ def read_write_and_read_case(self, file_path):
 
         return (original, reread)
 
+    @unittest.skip("bad test due to nan on missing entries")
```

Comment:
Is this meant to still be skipped?

Reply:
yes

Reply:
There is not much point replacing the old cases with new ones that the test fails on, then skipping the test. So I will revert the changes to this test. The "readability" of the new files is checked anyway in `test_read_case_pglib`.